k8s/configmap.yaml

```diff
@@ -2,18 +2,25 @@ apiVersion: v1
 kind: ConfigMap
 data:
   SCALYR_K8S_CLUSTER_NAME: <your-cluster-name>
-  SCALYR_K8S_DISABLE_API_SERVER: "true"
-  SCALYR_K8S_ALWAYS_USE_DOCKER: "true"
-  SCALYR_K8S_CACHE_EXPIRY_SECS: "600"
-  SCALYR_K8S_CACHE_START_FUZZ_SECS: "10"
-  SCALYR_K8S_CACHE_EXPIRY_FUZZ_SECS: "15"
-  SCALYR_COMPRESSION_TYPE: bz2
-  K8S_EVENTS_DISABLE: "true"
+  SCALYR_K8S_RATELIMIT_CLUSTER_NUM_AGENTS: "1000"
+  SCALYR_K8S_RATELIMIT_CLUSTER_RPS_INIT: "40"
+  SCALYR_K8S_RATELIMIT_CLUSTER_RPS_MIN: "8"
+  SCALYR_K8S_RATELIMIT_CLUSTER_RPS_MAX: "200"
```

Comment:
This seems quite low?  Does this mean we can't get up to 20,000 rps on this cluster? (20 requests per second, per node).

Reply:
Correct. As discussed with @czerwingithub, we want to keep this very timid to be absolutely sure no other network artifacts might interact with even a modest additional load from our agents.  Additionally, he asked for the cache interval period to be 20 minutes.

Reply:
Yep, I agree with the initial rate, but the max rate of only 200?  That means each node can only make 1 request every 5 seconds? (or have I not understood the math?).

Reply:
Yes that's right and I think that's the idea .. We want to be super conservative at the risk of slow data refresh (which isn't too big a deal).

With my test cluster with each agent monitoring 15 pods, it takes only between 3-7 minutes to get all the initial batch of data. Then every 20 min the pods are refetched.  

I guess the one risk is if for some reason certain nodes have a lot of pods whereas others have very little.  That could potentially take very long. 

cc @czerwingithub

Reply:
Do we know what their max pods per node were? (I remember average being 15).

Reply:
Let's continue this discussion on slack